src/node_api.cc

```diff
@@ -328,12 +330,18 @@ class CallbackWrapperBase : public CallbackWrapper {
         v8::Local<v8::External>::Cast(
             _cbdata->GetInternalField(kInternalFieldIndex))->Value());
     v8::Isolate* isolate = _cbinfo.GetIsolate();
-    cb(v8impl::JsEnvFromV8Isolate(isolate), cbinfo_wrapper);
+    napi_env env = reinterpret_cast<napi_env>(
+        v8::Local<v8::External>::Cast(
+            _cbdata->GetInternalField(kEnvIndex))->Value());
```

Comment:
Oh, and above that, we retrieve `the napi_callback_info` the same way, with a `reinterpret_cast`. Should we be using `static_cast` in all places where we place values into and retrieve values out of internal fields?